### PR TITLE
refactor(property-id): change localstorage implementation

### PR DESCRIPTION
**Pull request by M-OC**
- separates various 'localstorage' methods into separate files. (This is responsible for most of the  file changes in this PR. Mostly updating paths in tests, adding index.js etc)
- creates new 'localstorage' utility methods for setting/getting 'propertyId'
- updates 'getPropertyId' slightly to use 'getContainer' method. This method will be utilized in a follow-up PR when getting/setting container.
- Introduces two new types: container & containerSummary

### CODE REVIEW

File: src/lib/get-property-id.js [hunk 1]
```diff
@@ -1,40 +1,50 @@
...
+
+      return container.id;
+    })
+    .catch(() => {
+      // doing nothing for now since there's no logging
+      return '';
+    });
+};
+
+export const getContainerSettings = () : any => {
```
Comment (M-OC) [inline src/lib/get-property-id.js]: This is why the "WIP" label was added.

File: src/lib/constants.js [hunk 2]
```diff
@@ -5,7 +5,9 @@ export default {
   'accessTokenUrl': 'https://www.paypal.com/muse/api/partner-token',
   'storage': {
     'paypalCrUser': 'paypal-cr-user',
-    'paypalCrCart': 'paypal-cr-cart'
+    'paypalCrCart': 'paypal-cr-cart',
+    'paypalCrPropId': 'paypal-cr-propid',
+    'paypalCrContainer': 'paypal-cr-container'
```
Comment (joshbeam) [inline src/lib/constants.js]: what's this key for?

Comment (M-OC) [reply src/lib/constants.js]: It _will_ be used for container. A few pieces of this are "WIP".

File: src/lib/constants.js [hunk 3]
```diff
@@ -5,7 +5,9 @@ export default {
   'accessTokenUrl': 'https://www.paypal.com/muse/api/partner-token',
   'storage': {
     'paypalCrUser': 'paypal-cr-user',
-    'paypalCrCart': 'paypal-cr-cart'
+    'paypalCrCart': 'paypal-cr-cart',
+    'paypalCrPropId': 'paypal-cr-propid',
```
Comment (songz) [inline src/lib/constants.js]: could we just do `propertyId` instead of `PropId`

**Pull request merged**